django_lightweight_queue/cron_scheduler.py

```diff
@@ -16,7 +16,7 @@
 CRON_QUEUE_NAME = 'cron_scheduler'
 
 
-class CronScheduler(multiprocessing.Process):
+class CronScheduler(threading.Thread):
```

Comment:
Did you consider running this separately via subprocess too?

Reply:
I did, but it didn't seem super worthwhile to me. I'd like to change how we're doing cron scheduling to use systemd timers anyway.